### ISSUE TITLE
Feature request: Webservice endpoints to list aggregators and their parameters

### ISSUE THREAD

**Issue by lcoulet** [kairosdb/kairosdb#136]
Instead of having every single client to maintain its own list of aggregators with their various parameters, KairosDB should be able to send the list of aggregators to the clients with their parameters and a description of the aggregator and the input parameters.

So the clients could use this webservice to build dynamically the UI and would not need to be maintained for a specific version of KairosDB.

**Comment by lcoulet** [comment]
Hi there, we are thinking about implementing this feature and contributing it to kairosDB. Any thoughts about it? 
Can we propose something that you could merge in future versions of kairosDB?

**Comment by brianhks** [comment]
Obsofreakinglootly.  This was the design all along.  If you notice each aggregator has a description built in as an annotation, the purpose was to expose it via an api call.  Figuring out the parameters may be a bit more difficult (ie sampling).

If you do it, I'll take it.  We just havn't had the time to get around to it.

**Comment by lcoulet** [comment]
Yes we noticed. We will propose you with something on the dicussion group, 
Loic
